src/ontology/mondo-edit.obo

```diff
@@ -321171,18 +321172,18 @@ property_value: IAO:0000589 "meningioma (disease)" xsd:string
 [Term]
 id: MONDO:0016643
 name: frontonasal dysplasia
-def: "Frontonasal dysplasia caused by mutations in the ALX3 gene, encoding homeobox protein aristaless-like 3. It is inherited in an autosomal recessive fashion." [NCIT:C129028]
+def: "A group of rare bone development disorders characterized by an array of abnormalities affecting the eyes, forehead, and nose, and linked to midfacial dysraphia. The clinical picture is highly variable, but the major findings include hypertelorism, a broad nasal root, a large and bifid nasal tip, and widow's peak. Occasionally, abnormalities can include accessory nasal tags, cleft lip, ocular abnormalities (coloboma, cataract, microphthalmia), conductive hearing loss, basal encephalocele and/or agenesis of the corpus callosum. Intellectual deficit is rare and more likely to occur in cases where hypertelorism is severe or where there is extra-cranial involvement." [NCIT:C129028]
```

Comment:
Could you please change the definition source to Orphanet: Orphanet:250

Reply:
Done.